dataland-community-manager/src/test/kotlin/org/dataland/datalandcommunitymanager/services/DataRequestAlterationManagerTest.kt

```diff
@@ -86,9 +86,9 @@ class DataRequestAlterationManagerTest {
         }
         `when`(
             dataRequestRepository.searchDataRequestEntity(
-                searchFilter = DataRequestsQueryFilter(
-                    metaData.dataType.value, "",
-                    RequestStatus.Open.name, null, metaData.reportingPeriod, metaData.companyId,
+                searchFilter = DataRequestsFilter(
+                    setOf(metaData.dataType), null, null, metaData.companyId, metaData.reportingPeriod,
+                    setOf(RequestStatus.Open), null,
```

Comment:
Again make sure that the change to the RequestStatus works.

Reply:
We never insert `null` into the SQL query. Again, this is the purpose of the `used*Filter` properties.

Reply:
This i did not fully understand. My initial comment was in regards to the removal of .name from RequestStatus, which as i have seen now is part of the DataRequestFilter. So this is fine.
Maybe we can clarify this in a call.

Reply:
ok